src/renderer/components/status-bar/status-bar-item-registrator.injectable.tsx

```diff
@@ -74,6 +74,7 @@ const toItemInjectableFor = (extension: LensRendererExtension, getRandomId: () =
       id,
 
       instantiate: () => ({
+        id: extension.sanitizedExtensionId,
```

Comment:
There's something wrong here. This makes all of the status bar items from extension to have same ID?

Reply:
If that's the intended behaviour, then ID is not correct term for it.

Reply:
Yes, indeed. It should be called an `origin` not `id`.